src/Number.php

```diff
@@ -185,6 +185,11 @@ public function toFloat(): float
         return $this->value->toFloat();
     }
 
+    public function inCents(): int
+    {
+        return (int) str_replace('.', '', $this->toString());
```

Comment:
Can't you just multiply the value by 100?

Reply:
Yes, I thought I'd updated that. Just pushed that. 👍🏻